src/store/modules/auth/__tests__/reducer.test.js

```diff
@@ -1,11 +1,23 @@
 import { STATUS_SUCCESS } from '../../async/constants'
-import { SIGN_IN } from '../actionsTypes'
+import { SIGN_IN, SIGN_OUT } from '../actionsTypes'
 import { default as reducer, initialState } from '../reducer'
 
+const testToken = 'testtokentesttoken'
+
 it('populates state with sign in payload on sign in success', () => {
-  const action = { type: `${SIGN_IN}/${STATUS_SUCCESS}`, payload: { token: 'testtokentesttoken' } }
+  const action = { type: `${SIGN_IN}/${STATUS_SUCCESS}`, payload: { token: testToken } }
 
   const newState = reducer(initialState, action)
 
   expect(newState).toHaveProperty('token', action.payload.token)
 })
+
+it('sets token to null on sign out', () => {
+  const signInSuccessAction = { type: `${SIGN_IN}/${STATUS_SUCCESS}`, payload: { token: testToken } }
+  const signOutAction = { type: SIGN_OUT }
+
+  const stateWithToken = reducer(initialState, signInSuccessAction)
```

Comment:
what's the reason for populating state with token via action instead of just setting it directly for this test? i would suggest not relying on other actions to test this particular one